source/jquery.flot.js

```diff
@@ -2416,8 +2416,8 @@ Licensed under the MIT license.
                 // make sure we got room for the bar on the dancing floor
                 var delta;
 
-                //update bar width if needed
-                if (series.datapoints && series.datapoints.points) {
+                // update bar width if needed
+                if (series.datapoints && series.datapoints.points && series.bars.width === 1) {
```

Comment:
To be clear, in the example I provided, I was couching that under your proposed change, where we wouldn't be calling computeBarWidth, if we had a non-default barWidth set in our options.

On that note, your code uses 'series.bars.width'. Is this correct? The listed option (and what is used elsewhere) is 'series.bars.barWidth'.

Here is an example of an NXG graph with waveforms of different dts, each displaying as bars:
![nxgwaveformgraph](https://user-images.githubusercontent.com/26874831/45327280-816e4780-b51c-11e8-9a63-711640eade95.png)

This is correct. It sounds as if you are trying to modify this particular behavior. Is that right?

Reply:
Ah, that graph is very useful. I didn't know that overlaying bar charts was a regular use-case. Also, 'series.bars.width' was definitely incorrect. I'm not sure how that slipped through. Nonetheless, I've committed a new change that hopefully addresses your concerns. Rather than modifying the existing behavior, I am suggesting a flag that allows us to skip the bar width computation entirely.